CONTRIBUTING.md

```diff
@@ -76,6 +82,12 @@ flake8
 black . && isort .
 ```
 
+To run tests with coverage:
```

Comment:
Can we move this to `test-data/unit/README.md`?

CONTRIBUTING.md is aimed at newer contributors; I don't think many of them will be running coverage